src/models/Graph/convertFlyteGraphToDAG.ts

```diff
@@ -75,7 +74,7 @@ export function convertFlyteGraphToDAG(
 
     // Filter out any nodes with no parents (except for the start node)
     const result = values(nodeMap).filter(
-        n => n.id === nodeIds.start || (n.parentIds && n.parentIds.length > 0)
+        n => n.id === startNodeId || (n.parentIds && n.parentIds.length > 0)
```

Comment:
**nodeIds.start** was using same string to compare, replaced with startNodeId for consistency with other code.